cli/Makefile

```diff
@@ -22,6 +22,9 @@ GENERATED_FILES = internal/turbodprotocol/turbod.pb.go internal/turbodprotocol/t
 turbo: $(GENERATED_FILES) $(SRC_FILES) go.mod
 	CGO_ENABLED=1 go build $(GO_FLAGS) ./cmd/turbo
 
+shim: libturbo.a
```

Comment:
We should probably remove this because we now trigger the makefile from `build.rs`, so this will cause redundant building

Reply:
Since we're calling out to `make libturbo.a` we end up skipping `shim` here so it works out. I'll leave it for now as "safe dead code" but once I'm more-versed I'm pretty sure I agree that it should go.